tests/test_other.py

```diff
@@ -7984,33 +7984,33 @@ def run(*args):
 
     # test on libc++: see effects of emulated function pointers
     if self.is_wasm_backend():
-      run(['-O2'], 32, [], ['waka'], 226582,  20,  34, 561) # noqa
+      run(['-O2'], 34, [], ['waka'], 226582,  20,  34, 561) # noqa
     else:
-      run(['-O2'], 34, ['abort'], ['waka'], 186423,  28,  37, 537) # noqa
+      run(['-O2'], 34, ['abort'], ['waka'], 186423,  29,  38, 539) # noqa
       run(['-O2', '-s', 'EMULATED_FUNCTION_POINTERS=1'],
-                   34, ['abort'], ['waka'], 186423,  28,  38, 518) # noqa
+                   34, ['abort'], ['waka'], 186423,  29,  39, 519) # noqa
 
   def test_binaryen_metadce_hello(self):
     def run(*args):
       self.run_metadce_test(path_from_root('tests', 'hello_world.cpp'), *args)
 
     if self.is_wasm_backend():
-      run([],      16, [], ['waka'], 22185, 10,  17, 55) # noqa
-      run(['-O1'], 14, [], ['waka'], 10415,  8,  14, 29) # noqa
-      run(['-O2'], 14, [], ['waka'], 10183,  8,  14, 24) # noqa
-      run(['-O3'],  5, [], [],        2353,  7,   3, 14) # noqa; in -O3, -Os and -Oz we metadce
-      run(['-Os'],  5, [], [],        2310,  7,   3, 15) # noqa
-      run(['-Oz'],  5, [], [],        2272,  7,   2, 14) # noqa
+      run([],      23, [], ['waka'], 22185, 24,  17, 57) # noqa
+      run(['-O1'], 15, [], ['waka'], 10415, 15,  14, 33) # noqa
+      run(['-O2'], 15, [], ['waka'], 10183, 15,  14, 28) # noqa
+      run(['-O3'],  6, [], [],        2353,  9,   2, 15) # noqa; in -O3, -Os and -Oz we metadce
+      run(['-Os'],  6, [], [],        2310,  9,   2, 17) # noqa
+      run(['-Oz'],  6, [], [],        2272,  9,   2, 16) # noqa
```

Comment:
Nothing to do with this CL, but when changes break the metadce tests (which they often do) its quite hard to see whats changing.   Perhaps we should generate an "out" file containing all of the imports and exports by name rather than just number of them?

Reply:
In higher optimization levels those import names are minified, though, so it may not be feasible to have useful names for them.